SPECS/ca-certificates/ca-certificates.spec

```diff
@@ -425,6 +425,9 @@ rm -f %{pkidir}/tls/certs/*.{0,pem}
 %{_bindir}/bundle2pem.sh
 
 %changelog
+* Tue Nov 10 2020 Pawel Winogrodzki <pawelwi@microsoft.com> - 20200720-10
```

Comment:
Change to today's date?

Reply:
Nah, the change is actually from that date, it's just been sitting here a while, but I think it's better to say when this change was done instead of when we approved and merged the PR. Plus, I don't want to modify it for the sake of claiming, that the PR has been created in a fully automated way.:)